src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For instance, you can specify a generic font family like 'sans-serif' as a fallback.